.ci/install.sh

```diff
@@ -12,6 +12,8 @@ else
   exit 1
 fi
 
+INSTALL_DIR=${INSTALL_DIR:-$HOME}
```

Comment:
Installing to a subdirectory of the provided INSTALL_DIR sounds counter-intuitive to me. I think I'd rather have `INSTALL_DIR=$(pwd)/.fnm (curl ... | bash)` than the other way around. What do you think?

Reply:
Also, I would highly reconsider the usage of environment variables here.

I can suggest using options, [like RVM](https://rvm.io/rvm/install):

```bash
curl ... | bash                              # regular installation
curl ... | bash -s --install-dir $(pwd)/.fnm # install to specific directory
```

Reply:
I'm good with either. But it would complicate the install script.  Just let me know which you would rather have and I can make the change.

Reply:
It does complicate things a bit, but can still be manageable with proper writing (I dislike long bash scripts for that particular reason). Maybe something like

```bash
# at top, all the defaults

INSTALL_DIR=$HOME/.fnm

parse_args() {
  while [[ $# -gt 0 ]]; do
    key=$1
    case $key in
      --install-dir)
        $INSTALL_DIR=$2
        shift
        shift
      ;;
    esac
  done
}
```

kinda readable 🤷‍♂️ what do you think?